flight_phase_files/HALO_RF13_20200213_info.yaml

```diff
@@ -25,59 +30,326 @@ remarks:
 - During the whole flight precipitating cells were observed.
 - On the way to NTAS a few cold pools were observed with shallow sugar-type convection.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-02-13 08:20:47
-  end: 2020-02-13 09:16:21
-- kind: circle_break
-  name: cb1to2
-  start: 2020-02-13 09:16:21
-  end: 2020-02-13 09:29:33
-- kind: circle
-  name: c2
-  start: 2020-02-13 09:29:33
-  end: 2020-02-13 10:19:35
-- kind: circle_break
-  name: cb2to3
-  start: 2020-02-13 10:19:35
-  end: 2020-02-13 10:35:00
-- kind: circle
-  name: c3
-  start: 2020-02-13 10:35:00
-  end: 2020-02-13 11:27:10
-- kind: straight_leg
-  name: leg1_outbound_NTAS
-  start: 2020-02-13 11:34:30
-  end: 2020-02-13 12:05:48
-- kind: radar_calibration_wiggle
-  name: radar_cal_wiggle
+- kinds:
+  - circling
+  name: circling 1
+  irregularities: []
+  segment_id: HALO-0213_o1
+  start: 2020-02-13 08:19:47
+  end: 2020-02-13 11:31:01
+  dropsondes:
+    GOOD:
+    - HALO-0213_s01
+    - HALO-0213_s02
+    - HALO-0213_s03
+    - HALO-0213_s04
+    - HALO-0213_s05
+    - HALO-0213_s06
+    - HALO-0213_s07
+    - HALO-0213_s08
+    - HALO-0213_s09
+    - HALO-0213_s10
+    - HALO-0213_s11
+    - HALO-0213_s12
+    - HALO-0213_s14
+    - HALO-0213_s15
+    - HALO-0213_s16
+    - HALO-0213_s17
+    - HALO-0213_s18
+    - HALO-0213_s20
+    - HALO-0213_s21
+    - HALO-0213_s22
+    - HALO-0213_s23
+    - HALO-0213_s24
+    - HALO-0213_s25
+    - HALO-0213_s26
+    - HALO-0213_s27
+    - HALO-0213_s28
+    - HALO-0213_s29
+    - HALO-0213_s30
+    - HALO-0213_s31
+    - HALO-0213_s32
+    - HALO-0213_s33
+    - HALO-0213_s34
+    - HALO-0213_s35
+    - HALO-0213_s36
+    BAD:
+    - HALO-0213_s13
+    UGLY:
+    - HALO-0213_s19
+- kinds:
+  - circle
+  name: circle 1
+  irregularities: []
+  segment_id: HALO-0213_c1
+  start: 2020-02-13 08:19:47
+  end: 2020-02-13 09:15:39
+  dropsondes:
+    GOOD:
+    - HALO-0213_s01
+    - HALO-0213_s02
+    - HALO-0213_s03
+    - HALO-0213_s04
+    - HALO-0213_s05
+    - HALO-0213_s06
+    - HALO-0213_s07
+    - HALO-0213_s08
+    - HALO-0213_s09
+    - HALO-0213_s10
+    - HALO-0213_s11
+    - HALO-0213_s12
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle_break
+  name: circle break 1
+  irregularities: []
+  segment_id: HALO-0213_cb1
+  start: 2020-02-13 09:15:39
+  end: 2020-02-13 09:28:31
+  dropsondes:
+    GOOD: []
+    BAD:
+    - HALO-0213_s13
```

Comment:
After discussion of Geet and Tobi on dropsondes in circle breaks, it seems consequent to also move this BAD sonde to the circle 2.

Reply:
I moved the bad sondes from the circle break to circle 2 and started the circle 60s before that sonde.